src/neo/Consensus/ConsensusService.cs

```diff
@@ -629,7 +736,7 @@ private void RequestChangeView(ChangeViewReason reason)
         private bool ReverifyAndProcessPayload(ConsensusPayload payload)
         {
             if (!payload.Verify(context.Snapshot)) return false;
-            OnConsensusPayload(payload);
+            OnConsensusPayload(payload, true);
```

Comment:
Is this the only case where `avoidSaving` is set to `true`?

Reply:
Yes, what do you suggest?

Reply:
We can send only the payloads of the current height.

Reply:
Good idea, I think it will be almost the same and more simpler.

Done.